webpack.config.js

```diff
@@ -154,6 +154,12 @@ if ( isDevelopment || isDesktop ) {
 const cssFilename = cssNameFromFilename( outputFilename );
 const cssChunkFilename = cssNameFromFilename( outputChunkFilename );
 
+const fileLoader = FileConfig.loader( {
+	outputPath: path.join( __dirname, 'public', 'images' ),
+	publicPath: '../images/',
+	emitFile: browserslistEnv === 'evergreen', // Only output files once.
+} );
```

Comment:
This doesn't seem related?

Reply:
Indeed, that commit wasn't intended to be part of this PR. Rolled it back, thanks 🙇